lib/Retcon/DataSource.hs

```diff
@@ -130,12 +130,13 @@ createDocument src doc = do
 
     -- 4. Read handles
     output <- T.filter (not. isSpace) . T.decodeUtf8 <$> (liftIO . BS.hGetContents $ hout)
+    err <- T.decodeUtf8 <$> (liftIO . BS.hGetContents $ herr)
 
     -- 5. Check return code, raising error if required.
     exit <- liftIO $ waitForProcess hproc
     liftIO . debugM logName $ "CREATE exit: " <> show exit
     case exit of
-        ExitFailure c -> throwError $ ForeignError c output
+        ExitFailure c -> throwError $ ForeignError c err
```

Comment:
Do we maybe want both, just in case the data source is stupid? I.e. ("\nstdout: " <> err <> "\nstderr: " <> output)